### PR TITLE
log-backup: remove the timezone from log-date

### LINKED ISSUE #36370: br restore point failed with restored-ts "2022-07-16 20:30:09.068 +0800 CST" which is from log metadata output
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
1. Run log backup
2. Run log metadata to get the range that can be recovered
```
[2022/07/16 21:38:18.047 +08:00] [INFO] [collector.go:69] ["log metadata"] [log-min-ts=434606285735330080] [log-min-date="2022-07-15 21:23:29.818 +0800 CST"] [log-max-ts=434628095919522279] [log-max-date="2022-07-16 20:30:09.068 +0800 CST"]
```

### 2. What did you expect to see? (Required)
Restore should succeed

### 3. What did you see instead (Required)
Restore failed 
```
tiup br:nightly log truncate -s "s3://xxx" --until "2022-07-16 20:30:09.068 +0800 CST"
...
Detail BR log in /tmp/br.log.2022-07-16T21.49.11+0800
Error: must set timezone when using datetime format ts
```

### 4. What is your TiDB version? (Required)
6.2 nightly

### PR DESCRIPTION
**Pull request by MoCuishle28**
### What problem does this PR solve?

Issue Number: close #36370

Problem Summary:

### What is changed and how it works?
Remove the timezone info from log-date.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Manual Test
- enter `br log status -u {PD-IP}:{PD-Port}`
- enter `br log metadata -s 's3://{bucket}/{prefix}'`

output:
```
● Total 1 Tasks.
> #1 <
              name: pitr
            status: ● NORMAL
             start: 2022-07-20 15:43:43.178 +0800
               end: 2090-11-18 22:07:45.624 +0800
           storage: s3://{bucket}/{prefix}
       speed(est.): 0.00 ops/s
checkpoint[global]: 2022-07-20 16:28:18.378 +0800; gap=22s
```

```
[2022/07/20 16:33:00.737 +08:00] [INFO] [collector.go:69] ["log metadata"] [log-min-ts=434604259287760897] [log-min-date="2022-07-15 19:14:39.534 +0800"] [log-max-ts=434605479096221697] [log-max-date="2022-07-15 20:32:12.734 +0800"]
[2022/07/20 16:33:00.737 +08:00] [INFO] [collector.go:69] ["log metadata success summary"] [total-ranges=0] [ranges-succeed=0] [ranges-failed=0] [total-take=136.799319ms]
```

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by joccau — APPROVED** [review]
LGTM

File: br/pkg/stream/stream_status.go [hunk 1]
```diff
@@ -117,9 +117,9 @@ func (p *printByTable) AddTask(task TaskStatus) {
 	table := p.console.CreateTable()
 	table.Add("name", task.Info.Name)
 	table.Add("status", task.colorfulStatusString())
-	table.Add("start", fmt.Sprint(oracle.GetTimeFromTS(task.Info.StartTs)))
+	table.Add("start", fmt.Sprint(oracle.GetTimeFromTS(task.Info.StartTs).Format("2006-01-02 15:04:05.999999999 -0700")))
```
Comment (3pointer) [inline br/pkg/stream/stream_status.go:120]: maybe use a common function and add a unit test case to make sure the output of this common function.

Comment (MoCuishle28) [reply br/pkg/stream/stream_status.go]: Done. PTAL

**Review by 3pointer — APPROVED** [review]
LGTM

**Pull request merged**